config/routes.rb

```diff
@@ -13,4 +13,6 @@
   get "/404", to: "errors#not_found", via: :all
   get "/422", to: "errors#unprocessable_entity", via: :all
   get "/500", to: "errors#internal_server_error", via: :all
+
+  get "/courses/:provider_code/:course_code", to: "courses#show", as: "course"
```

Comment:
is that right 

https://find-postgraduate-teacher-training.education.gov.uk/course/T92/X130